types/board_support_common.yaml

```diff
@@ -0,0 +1,54 @@
+description: EV and EVSE board support types
+types:
+  BspEvent:
+    description: Event stream from ControlPilot signal/Relais
+    type: object
+    additionalProperties: false
+    required:
+      - event
+    properties:
+      event:
+        type: string
+        enum:
+          - 'A'
+          - 'B'
+          - 'C'
+          - 'D'
+          - 'E'
+          - 'F'
+          - 'PowerOn'
+          - 'PowerOff'
+          - 'ErrorDF'
+          - 'ErrorRelais'
+          - 'ErrorVentilationNotAvailable'
+          - 'ErrorOverCurrent'
+          - 'ErrorOverVoltage'
+          - 'ErrorUnderVoltage'
+          - 'ErrorMotorLock'
+          - 'ErrorOverTemperature'
+          - 'ErrorBrownOut'
+          - 'ErrorCablePP'
+          - 'ErrorEnergyManagement'
+          - 'ErrorNeutralPEN'
+          - 'ErrorCpDriver'
```

Comment:
Should 'ErrorCpDriver' be sent in case of an invalid CP state?

Reply:
This has been replaced with an MREC code with the following definition: "MREC_14_PilotFault: The control pilot voltage is out of range."